### ISSUE TITLE
Legend (Feature Request?)

### ISSUE THREAD

**Issue by ck9ine** [blocklist-tools/blocklist-tools#3]
How does one properly interpret the ouutput of the blocklist tool?  I'm sure many find it self explanatory, but for some reason, my mind seems to think it a little ambiguous with no example or symbol legend.  :/ 

Sorry, couldn't find mention of the meanings of the green "check mark" and red "not sign" symbols in any documentation.

"Blocklists that contain a matching entry, including the date it was first added, and optionally removed."  

Does this mean there will be two dates if a matching entry and is entered and removed?  Does the red "not sign" mean that it was originally in the list, but removed.

How does "Initial Version" and "Current Version" fit in with the meanings behind the symbos as well as presented date?

**Comment by lightswitch05** [comment]
Great questions! Pull requests are welcome for a better design ;). It was rushed cause I had a newborn on the way and knew I didn't have much time to put into the project. In fact, I'm rocking her now as I answer your question.

Anyways, green means currently blocked. Red means no longer blocked. First date is the date it was first blocked. Second date means the date it was last removed from the list.

I hope that helps!

**Comment by ck9ine** [comment]
Sorry for the late reply. I've been busy as well.  Congrats on your baby, and really appreciate that the tool was created in the first place, especially now knowing how busy you were/are.  :) I'm new to the ways of github, so I'm not sure what a pull request is, and I may not have the skills to assist (though once upon a time I did do some very simple basic web dev), but I will look into it when I have some time.  

To be honest, your provided description is really good.  On first read, it was a little confusing, but when I pulled up an example domain, say "unagi.amazon.com", I realized how clear and concise it was.  :)  Perhaps the 3 paragraphs can simply be added as a small footnote at the very bottom of the output page?  Starting with "...Green means currently blocked" to "then it would not be in the results list."

Thanks again for all your hard work!
